packages/consumption/test/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.test.ts

```diff
@@ -383,7 +383,7 @@ describe("ProposeAttributeRequestItemProcessor", function () {
             const result = await processor.canAccept(requestItem, acceptParams, request);
 
             expect(result).errorValidationResult({
-                code: "error.consumption.requests.invalidRequestItem",
+                code: "error.consumption.requests.invalidAcceptParameters",
                 message: /The given Attribute belongs to someone else. You can only share own Attributes./
```

Comment:
What's the reason you're using a regex here? If you want an exact match, use a string instead.

Reply:
The Error messages were already there as a regex and not as a string. I also noticed this when I was working with the processors in connection with the validation of Requests. I would also prefer to use strings instead of regex. Could rewrite it in PR https://github.com/nmshd/runtime/pull/32 for the ProposeAttributeRequestItemProcessor and the ReadAttributeRequestItemProcessor, I worked a lot on the tests and errors for them there anyway.

Reply:
Fine by me 👍